src/test/java/org/jscep/message/PkiMessageEncoderTest.java

```diff
@@ -130,6 +130,32 @@ public void simpleTest() throws Exception {
 
         assertEquals(message, actual);
     }
+    
+    @Test
+    public void simpleTestTripleDES() throws Exception {
+        KeyPair caPair = KeyPairGenerator.getInstance("RSA").generateKeyPair();
+        X509Certificate ca = X509Certificates.createEphemeral(
+                new X500Principal("CN=CA"), caPair);
+
+        KeyPair clientPair = KeyPairGenerator.getInstance("RSA")
+                .generateKeyPair();
+        X509Certificate client = X509Certificates.createEphemeral(
+                new X500Principal("CN=Client"), clientPair);
+
+        // Everything below this line only available to client
+        PkcsPkiEnvelopeEncoder envEncoder = new PkcsPkiEnvelopeEncoder(ca,
+                "DES3");
```

Comment:
This isn't in the Java Standard Algorithm Names; can you consider using `DESede` instead please?